include/call.hpp

```diff
@@ -322,7 +322,7 @@ class call : virtual public task, virtual public listener, public virtual socket
     BIO *m_bio;
 #endif
 
-    int _callDebug(const char *fmt, ...);
+    int _callDebug(const char *fmt, ...) __attribute__((format(printf, 2, 0)));
```

Comment:
No, there's an implicit 'this' argument that gcc counts. I thought so too but got a nice warning.